dashboard/src/modules/components/DatePickerComponent/index.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from "react";
+import {
+  InputGroup,
+  InputGroupText,
+  DatePicker,
+  isValidDate,
+  Button,
+} from "@patternfly/react-core";
+import "./index.css";
+import { formatDate } from "../../../utils/dateFormatter";
+import { filterData } from "../../../utils/filterDataset";
+import { constructUTCDate } from "../../../utils/constructDate";
+import bumpToDate from "utils/bumpDate";
+import addOneDay from "utils/addDay";
+
+function DatePickerWidget({
+  dataArray,
+  setPublicData,
+  controllerName,
+  setDateRange,
+}) {
+  const [fromDate, setFromDate] = useState({});
+  const [toDate, setToDate] = useState(
+    constructUTCDate(formatDate(new Date(bumpToDate(new Date(), 1))))
+  );
+  const [strDate, setStrDate] = useState(
+    new Date().toLocaleString("fr-CA").split(",")[0]
```

Comment:
Do we really want to hardcode the dashboard to `fr-CA` (French Canadian) localized date!?? (And that's a relatively interesting choice anyway.) I assume this is probably "YYYY-MM-DD" format since they tend to be a bit more "European" than Americans, and if so I prefer that format ... but it still seems a bit awkward to hardcode this!

Reply:
I googled and found this as the easisest method to convert a date into YYYY-MM-DD :)

Reply:
I've done some looking, too, and I haven't yet found a good way to address this.  The locale functions take an optional locale first (omitting it defaults to the browser's choice), and _then_ options...so we can't customize the output until after we specify what we're departing from...and, I don't see a "generic" (a la "C") locale that we could specify.

So, the prevailing wisdom is to just pick a locale and then, if needed, customize it...but, `fr-CA` (or `sv`) doesn't require additional customization here, so it's fine...but, it would be good to have a comment explaining this....

Reply:
Yes, please; a line comment for strange things like this is always a good idea!

Reply:
@dbutenhof Would a comment like **Converting date object to YYYY-MM-DD string** would work?

Reply:
I'm more interested in documenting the specific choice of the `fr_CA` locale rather than the intent of the conversion; but, "that, too".

Maybe `// specify fr-CA locale to be sure the date is formatted as standard YYYY-MM-DD`??